Code/Framework/AzFramework/AzFramework/Spawnable/SpawnableSystemComponent.cpp

```diff
@@ -20,11 +21,27 @@ namespace AzFramework
     {
         Spawnable::Reflect(context);
         SpawnableMetaData::Reflect(context);
+        EntitySpawnTicket::Reflect(context);
 
         if (auto* serializeContext = azrtti_cast<AZ::SerializeContext*>(context); serializeContext != nullptr)
         {
             serializeContext->Class<SpawnableSystemComponent, AZ::Component>();
             serializeContext->RegisterGenericType<AZ::Data::Asset<Spawnable>>();
+
+            if (AZ::EditContext* editContext = serializeContext->GetEditContext())
+            {
+                editContext->Class<EntitySpawnTicket>("EntitySpawnTicket", "An object used to spawn prefabs with SpawnableEntitiesInterface.");
```

Comment:
This could use a bit of expansion, at least to indicate that there's a reference to a spawnable asset and the entities spawned using it and that the lifecycle is tied to this ticket.

Reply:
I wasn't sure whether to write a long explanation in tooltip of a short summary?

Reply:
Same recommendation as earlier : "'EntitySpawnTicket is an object used to identify and track the spawned entities associated with the ticket'". 
I also thought about 'manages the lifecycle of entities' but thought that's a bit of a stretch because the entity can be destroyed even without the ticket.